README.md

```diff
@@ -195,6 +198,8 @@ require'nvim-treesitter.configs'.setup {
       keymaps = {
         goto_definition = "gnd",
         list_definitions = "gnD",
+        goto_next_usage = "<a-*>",
+        goto_previous_usage = "<a-#>",
```

Comment:
Can use use `Shift` here instead ? just to be q bit less opinionated 😉

Reply:
On my German keyboard "*" already implies shift. I guess also on US keyboards.

Reply:
Oh let's leave it like that then

Reply:
I really don't care about the defaults. Any suggestion is ok for me.

We're already using scheme-like syntax, don't you realize that nvim is slowly becoming emacs? We should  set <c-r><c-a><m-*> :stuck_out_tongue: